app/components/bs-alert-dismiss/component.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'span',
+  action: 'dismiss',
+
+  click: function(){
+    this.get('target').send('dismiss');
```

Comment:
correct — I  backed into this syntax because this sub-component needs its target explicitly set. I'm going to change it back.